connectorx/src/get_arrow2.rs

```diff
@@ -114,6 +115,36 @@ pub fn get_arrow2(
                     );
                     dispatcher.run()?;
                 }
+                ("simple", Some(tls_conn)) => {
+                    let sb = PostgresSource::<SimpleProtocol, MakeTlsConnector>::new(
+                        config,
+                        tls_conn,
+                        queries.len(),
+                    )?;
+                    let dispatcher = Dispatcher::<
+                        _,
+                        _,
+                        PostgresArrowTransport<SimpleProtocol, MakeTlsConnector>,
```

Comment:
This transport should be `PostgresArrow2Transport`, the same applies to the below one.

Reply:
fixed it